snippets/ModalDialog.md

```diff
@@ -0,0 +1,133 @@
+### Modal/Dialog
+
+Renders a dialog component with modal.
+
+This is a simple React ModalDialog Component.It owns 4 props: `show`、`close`、`title` and `clickModal2Hide`.
+
+* `show` prop determines the initial state of the dialog(show/hide).It can influence the `display` css property of `modal` class div.E
+* `close` prop defines the method for closing the dialog.
+* `title` prop determines the title of the dialog.
+* `clickModal2Hide` prop determines whether we can close the dialog via clicking on modal.
```

Comment:
This should probably be renamed to something that is easier to remember... Best I can come up with is `closeOnclick`
```suggestion
* `closeOnClick` prop determines whether we can close the dialog via clicking on modal.
```